.github/workflows/build.yml

```diff
@@ -1,36 +1,23 @@
-name: Build
+name: Build and Test
 on: [push, pull_request]
 jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
     steps:
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v1
-
       - name: Set up Go 1.13
         uses: actions/setup-go@v1
         with:
           go-version: 1.13
         id: go
 
-      - name: Cache (dependencies)
-        uses: actions/cache@v1
-        id: cache
-        with:
-          path: ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
-
-      - name: Install linter
-        run: "curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.22.2"
-
-      - name: Install dependencies
-        if: steps.cache.outputs.cache-hit != 'true'
-        run: go mod download
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
 
-      - name: Lint
+      - name: Build, run tests and vet
         run: |
-          export PATH=$PATH:~/go/bin
-          ./run_tests.sh lint
```

Comment:
With this change, we'd need to remove the `./run_tests.sh` file as well. However, I'd recommend keeping it. Should probably rename it though, to say `./lint.sh`.